source/extensions/omni.isaac.lab/omni/isaac/lab/utils/timer.py

```diff
@@ -60,14 +60,23 @@ class Timer(ContextDecorator):
     Reference: https://gist.github.com/sumeet/1123871
     """
 
-    def __init__(self, msg: str | None = None):
+    timing_info = dict()
+    """ Used to log timing instances for retrieval from outside
+        of the Timer class. Times will be logged if a non-empty
+        name is provided on stop.
+    """
```

Comment:
Maybe stating it as class-var explicitly makes it clearer: https://peps.python.org/pep-0526/#class-and-instance-variable-annotations

```suggestion
    timing_info: ClassVar[dict[str, float]] = dict()
    """Dictionary for storing the elapsed time per timer instances globally.
    
    This dictionary logs the timer information. The keys are the names given to the timer class
    at its initialization. If no :attr:`name` is passed to the constructor, no time
    is recorded in the dictionary.
    """
```